hardhat/tasks/sips/args/sipArgs.js

```diff
@@ -1104,6 +1104,159 @@ const getArgsSIP0077 = async (hre) => {
     return { args, governor: "GovernorAdmin" };
 };
 
+const getArgsSip0080 = async (hre) => {
```

Comment:
```suggestion
const getArgsSip0084 = async (hre) => {
```
also rename the file and the Description text

Reply:
Updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/549/commits/d47e661d64be04ec9d03dbffbe6d7b0dd835ca50)